samples/go/csv/read.go

```diff
@@ -157,18 +157,34 @@ func readFieldsFromRow(row []string, headers []string, fieldOrder []int, kindMap
 	return fields
 }
 
+func primaryKeyFromFields(fields types.ValueSlice, fieldOrder, pkIndices []int) (types.ValueSlice, types.Value) {
```

Comment:
Can you add a comment? e.g. why is a primary key multiple returned values? what do the different return values represent?